test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/UsedAllArchivedLambdas.java

```diff
@@ -66,7 +66,7 @@ static void test() throws Exception {
             "-Xlog:cds=debug",
             "-cp", appJar, mainClass, "run")
             .assertNormalExit(output -> {
-                output.shouldContain("Used all archived lambda proxy classes for: UsedAllArchivedLambdasApp run()Ljava/lang/Runnable;")
+                output.shouldContain("Used all dynamic archived lambda proxy classes for: UsedAllArchivedLambdasApp run()Ljava/lang/Runnable;")
```

Comment:
The "Used all static archived lambda proxy classes for" log output isn't applicable to the `UsedAllArchivedLambdas.java` test because the lambda proxy classes are all in the dynamic archive. It could be applicable to the new test. I'll check.

Reply:
@ashu-mehra Could you review my updated changes? Thanks.

Reply:
Looks fine.